src/components/Button/Button.js

```diff
@@ -1,9 +1,29 @@
 import React from 'react';
 import './Button.scss';
 
-const ButtonCompornent = (props) => {
-return (
-<button></button>;
-)};
+const Button = (props) => {
+  const {
+    type,
+    value,
+    name,
+    size,
+    shape,
+    disabled,
+    onclick,
+    className = 'btn',
+  } = props;
+  return (
+    <button
+      type={type}
+      value={value}
+      name={name}
+      size={size}
+      shape={shape}
+      disabled={disabled}
+      onclick={onclick}
```

Comment:
```suggestion
      onClick={onclick}
```
onClick이벤트입니다! 대소문자 구분에 주의 해주세요!

Reply:
피드백반영하였습니다.